Sources/PrimerSDK/Classes/User Interface/TokenizationViewModels/CheckoutWithVaultedPaymentMethodViewModel.swift

```diff
@@ -196,14 +204,20 @@ class CheckoutWithVaultedPaymentMethodViewModel: LogReporter {
 
                 Timer.scheduledTimer(withTimeInterval: 5, repeats: false) { [weak self] _ in
                     if !decisionHandlerHasBeenCalled {
-                        self?.logger.warn(message: "The 'decisionHandler' of 'primerHeadlessUniversalCheckoutWillCreatePaymentWithData' hasn't been called. Make sure you call the decision handler otherwise the SDK will hang.")
+                        let message =
+"""
+The 'decisionHandler' of 'primerHeadlessUniversalCheckoutWillCreatePaymentWithData' hasn't been called.
+Make sure you call the decision handler otherwise the SDK will hang.
+"""
```

Comment:
I think @jnewc flagged this elsewhere, but we are introducing a newLine here in the output which is not what we are after.

Reply:
The following would maintain the same output:

```swift
"""
The 'decisionHandler' of 'primerHeadlessUniversalCheckoutWillCreatePaymentWithData' hasn't been called.\
Make sure you call the decision handler otherwise the SDK will hang.
"""
```

Reply:
Changed on all occurrences in the code. Thanks!